.travis.yml

```diff
@@ -4,6 +4,8 @@ jdk: oraclejdk8
 
 env:
   global:
+    - SONAR_HOST_URL="https://sonarcloud.io"
+#TODO    - SONAR_LOGIN=
```

Comment:
@KengoTODA you have already added SpotBugs project to sonarcloud.io, so could you please give me a sonar login token as a secret prepared with Travis?